wiki/Tournaments/CFB/5/en.md

```diff
@@ -0,0 +1,171 @@
+| Eliminated | ![][flag_FR] [SverdWyrd](https://osu.ppy.sh/users/10996443), ![][flag_FR] [Hitsumo](https://osu.ppy.sh/users/3199415) |
+
+*Note: Players are sorted by the Qualifiers results.*
+
+## Mappools
+
+### Round of 16
+
+**[Download the mappack here! (80 MB)](https://mega.nz/file/xpkkFQIL#r7YabtaKEqGcJSRZfhhD-lsvPengCiaNI88th-WfglU)**
+
+- NoMod
+  - [senya - Zetsubou no Fuchi (Ascendance) \[IDu5t's Platter\]](https://osu.ppy.sh/beatmapsets/762829#fruits/1871747)
+  - [Zektbach - Turii \~Panta rhei\~ (Kyuare) \[Platter\]](https://osu.ppy.sh/beatmapsets/685428#fruits/1567499)
+  - [Fox Stevenson - Flash (\_orange) \[Light Insane\]](https://osu.ppy.sh/beatmapsets/649081#fruits/1421055)
+  - [Silentroom - Protoflicker (DeRandom Otaku) \[Xenon's Insane\]](https://osu.ppy.sh/beatmapsets/1016347#fruits/2145994)
+- Hidden
+  - [nano.RIPE - Kodama Kotodama (CLSW) \[Rain\]](https://osu.ppy.sh/beatmapsets/340570#fruits/753305)
+  - [Suzuki Masayuki - DADDY ! DADDY ! DO ! feat. Suzuki Airi (TV Size) (hypercyte) \[Insane?\]](https://osu.ppy.sh/beatmapsets/1147361#fruits/2395929)
+- HardRock
+  - [IKIMONOGAKARI - Hotaru no Hikari (TV Size) (Secre) \[Chunin\]](https://osu.ppy.sh/beatmapsets/1198625#fruits/2496371)
+  - [kradness&Reol - Okochama Sensou (Kamio Misuzu) \[wMw's Advanced\]](https://osu.ppy.sh/beatmapsets/369058#fruits/847908)
+- DoubleTime
+  - [M2U - Marigold (feat. Guriri) (Hareimu) \[Salad\]](https://osu.ppy.sh/beatmapsets/596327#fruits/1262956)
+  - [UPLIFT SPICE - Omega Rhythm (Jemmmmy) \[Hard\]](https://osu.ppy.sh/beatmapsets/147962#fruits/534080)
+- Tiebreaker
+  - **[Virtual Self - Angel Voices (Sinnoh) \[Platter\]](https://osu.ppy.sh/beatmapsets/939437#fruits/2060158)**
+
+### Qualifiers
+
+- NoMod
+  - [Colors\*Slash - Colors Power ni Omakasero! (Sober Bear Remix) (Jemzuu) \[Greaper's Platter\]](https://osu.ppy.sh/beatmapsets/1213101#fruits/2545167)
+  - [nmk - sola (sjoy) \[cwR's Insane\]](https://osu.ppy.sh/beatmapsets/183267#fruits/440413)
+- Hidden
+  - [t+pazolite - CENSORED!! (Fanteer) \[Hyper\]](https://osu.ppy.sh/beatmapsets/501407#fruits/1072328)
+- HardRock
+  - [Tanya Degurechaff (CV: Yuuki Aoi) - Los! Los! Los! (TV Size) (Du5t) \[Private\]](https://osu.ppy.sh/beatmapsets/904574#fruits/1888832)
+- DoubleTime
+  - [antiPLUR - One Life Left to Live (JBHyperion) \[Cup\]](https://osu.ppy.sh/beatmapsets/957842#fruits/2005438)
+
+## Match results
+
+### Qualifiers
+
+- More detailed results are available on the [statistics sheet](https://docs.google.com/spreadsheets/d/1jhNXMAqAxy5dwruO6uiC1VuW_Te-AyoVSgmeoJxB6cE/edit?usp=sharing)
+- Average rank is determined by `Σ position earned when playing the Qualifiers mappool / amount of Qualifiers mappool`
+- Only the first 3 categorised as top seed, since one of the participant was disqualified due to not met the screening phase criteria from osu! officials
```

Comment:
```suggestion
- Only the first 3 is categorised as top seed, since one of the participant was disqualified due to not met the screening phase criteria from osu! officials
```